Makefile

```diff
@@ -505,7 +505,7 @@ endif
 # Build and test addons without building anything else
 # Related CI job: node-test-commit-arm-fanned
 test-ci-native: LOGLEVEL := info
-test-ci-native: | test/addons/.buildstamp test/js-native-api/.buildstamp test/node-api/.buildstamp
+test-ci-native: | bench-addons-build test/addons/.buildstamp test/js-native-api/.buildstamp test/node-api/.buildstamp
```

Comment:
```suggestion
test-ci-native: | benchmark/napi/.buildstamp test/addons/.buildstamp test/js-native-api/.buildstamp test/node-api/.buildstamp
```
https://github.com/nodejs/node/blob/b0b52b2023f5cd0df4ae921850815586b4313dca/Makefile#L1178

Reply:
![I have no idea what I'm doing.](https://media.giphy.com/media/ToMjGpNuOksUiclTp4c/source.gif)

Reply:
In case anyone else is puzzled, `bench-addons-build` depends on `$(NODE_EXE)` which means it will attempt to build the Node.js binary (`$NODE_EXE` is a `.PHONY` target so is always executed: https://github.com/nodejs/node/blob/b0b52b2023f5cd0df4ae921850815586b4313dca/Makefile#L93). As the comment for `test-ci-native` says, we only want to build the addons so we should not be depending on the `bench-addons-build` target -- the test machines that we execute `test-ci-native` on are not set up to build the Node.js binary itself.